frontend/scripts/react-components/tool-layers/tool-layers.fetch.saga.js

```diff
@@ -119,3 +120,24 @@ export function* getMapDimensions(selectedContext, selectedYears) {
     }
   }
 }
+
+export function* getUnitLayerData(params) {
+  const { selectedGeoColumnId, selectedUnitIndicatorIds, iso2 } = params;
+  const url = `//p2cs-sei.carto.com/api/v2/sql?q=SELECT node_id, geo_id, attribute_id, json_object_agg(COALESCE(year, 0 ), total ORDER BY year) as years FROM (SELECT node_id, geo_id, attribute_id, year, SUM(value) AS total FROM map_attributes_values WHERE node_type_id = ${selectedGeoColumnId} and attribute_id IN (${selectedUnitIndicatorIds.join(',')}) and iso2 = '${iso2}' GROUP BY node_id, geo_id, attribute_id, year) s GROUP BY node_id, geo_id, attribute_id ORDER BY geo_id, node_id, attribute_id`;
```

Comment:
i recommend using something like `oneLineTrim template literal` from 'common-tags' so you can separate these into one line for easier readability https://www.npmjs.com/package/common-tags#onelinetrim 

@Bluesmile82